scripts/run-qemu-from-export

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script runs qemu and starts a VM using files included in the
+# rtos_nilinuxembedded exports. It will also build the qemu-helper-native recipe
+# if it has not already been built.
+# The script expects the path to the qemu directory in the rtos_nilinuxembedded
+# export as an argument.
+
+SCRIPT_ROOT=$(realpath $(dirname ${BASH_SOURCE:-${0}}))
+REPO_ROOT=$(dirname "$SCRIPT_ROOT")
+QEMU_FILES_PATH=$1
+
+git submodule update --init --recursive
+
+DOCKER_IMAGES="$(docker images build-nile)"
+echo $DOCKER_IMAGES
+if [[ !  $DOCKER_IMAGES =~ "build-nile" ]]; then
+	source $REPO_ROOT/docker/create-build-nile.sh
```

Comment:
Any reason this has to be `source ..` instead of `bash ..`?

Usually `source` is used if some variables/functions in that another file are used in this file and not just to execute a script.